semconv/src/main/java/io/opentelemetry/semconv/SchemaUrls.java

```diff
@@ -1,16 +0,0 @@
-/*
- * Copyright The OpenTelemetry Authors
- * SPDX-License-Identifier: Apache-2.0
- */
-
-package io.opentelemetry.semconv;
-
-public final class SchemaUrls {
```

Comment:
This is not generated. We manually add constants to it with each release. There's a build check it make sure it has variables for each verion.

Reply:
Fixed